views/templates/learner/feedback.tpl

```diff
@@ -0,0 +1,29 @@
+<?php
+use oat\tao\helpers\Template;
+?><!DOCTYPE html>
+<html>
+<head>
+    <title><?=__('Test error');?></title>
+    <link rel="stylesheet" href="<?= Template::css('reset.css','tao') ?>"/>
+    <link rel="stylesheet" href="<?= Template::css('custom-theme/jquery-ui-1.9.2.custom.css','tao') ?>"/>
+    <link rel="stylesheet" href="<?= Template::css('feedback.css') ?>"/>
+    <link rel="shortcut icon" href="<?= Template::img('favicon.ico', 'tao') ?>"/>
+</head>
+<body>
+<div id="main" class="ui-widget-content ui-corner-all">
+<?php if (isset($reason) && $reason == 'concurrent-test'): ?>
+    <h1>Test suspended</h1>
+    <div class="message">
+        <?= __('The test was suspended because another one was opened in a different window or tab'); ?>.
```

Comment:
- [ ] We need to generate translation strings for this files. @jsconan may help with this.